Configuration/StandardSequences/python/AlCaHarvesting_cff.py

```diff
@@ -88,34 +88,6 @@
                                               timetype   = cms.untracked.string('lumiid')
                                               )
 
-
-# --------------------------------------------------------------------------------------
-# BeamSpot HP - Low PU - by Run
-ALCAHARVESTBeamSpotHPLowPUByRun = ALCAHARVESTBeamSpotHPByRun.clone()
-ALCAHARVESTBeamSpotHPLowPUByRun.AlcaBeamSpotHarvesterParameters.BeamSpotModuleName = cms.untracked.string('alcaBeamSpotProducerHPLowPU')
-
-# configuration of DropBox metadata and DB output
-ALCAHARVESTBeamSpotHPLowPUByRun_metadata = cms.PSet(record = cms.untracked.string('BeamSpotObjectsRcdHPByRun'))
-
-ALCAHARVESTBeamSpotHPLowPUByRun_dbOutput = cms.PSet(record = cms.string('BeamSpotObjectsRcdHPByRun'),
-                                                    tag = cms.string('BeamSpotObjectHP_ByRun'),
-                                                    timetype   = cms.untracked.string('runnumber')
-                                                    )
-
-# --------------------------------------------------------------------------------------
-# BeamSpot HP - Low PU - by Lumi
-ALCAHARVESTBeamSpotHPLowPUByLumi = ALCAHARVESTBeamSpotHPByLumi.clone()
-ALCAHARVESTBeamSpotHPLowPUByLumi.AlcaBeamSpotHarvesterParameters.BeamSpotModuleName = cms.untracked.string('alcaBeamSpotProducerHPLowPU')
-
-
-# configuration of DropBox metadata and DB output
-ALCAHARVESTBeamSpotHPLowPUByLumi_metadata = cms.PSet(record = cms.untracked.string('BeamSpotObjectsRcdHPByLumi'))
-
-ALCAHARVESTBeamSpotHPLowPUByLumi_dbOutput = cms.PSet(record = cms.string('BeamSpotObjectsRcdHPByLumi'),
-                                                     tag = cms.string('BeamSpotObjectHP_ByLumi'),
-                                                     timetype   = cms.untracked.string('lumiid')
-                                                     )
-
```

Comment:
@tocheng why are you removing this recent addition? is this intended, and if yes why, or is it just by accident?